R/tm_a_pca.R

```diff
@@ -25,14 +25,13 @@
 #' # ADSL example
 #'
 #' library(nestcolor)
-#' library(scda)
-#' ADSL <- synthetic_cdisc_data("latest")$adsl
+#' ADSL <- rADSL
```

Comment:
```suggestion
#' ADSL <- teal.modules.general::rADSL
```
Can you put name of the package before the name of the dataset so it is clear it comes from this package? I found rADSL data in other packages, like teal.transform. Can you apply this comment to all examples in all R files that you modified?

Reply:
Done! https://github.com/insightsengineering/teal.modules.general/pull/534/commits/8317d53b0b1d7971eed34f51c03296677eb4746a